app/move/steps/update.js

```diff
@@ -0,0 +1,43 @@
+const { cloneDeep } = require('lodash')
+
+const {
+  Assessment,
+  MoveDetails,
+  PersonalDetails,
+} = require('../controllers/update')
+
+const createSteps = require('./create')
+
+const updateControllers = {
+  '/personal-details': PersonalDetails,
+  '/move-details': MoveDetails,
+  '/court-information': Assessment,
+  '/risk-information': Assessment,
+  '/health-information': Assessment,
+}
+
+const provideStepProps = (key, step) => {
+  const stepProps = {
+    entryPoint: true,
+    checkSession: false,
+    checkJourney: false,
+    update: true,
+    updateBackStep: '/move/:moveId',
```

Comment:
Could we make use of the existing properties that exist? Like `backLink`?

Reply:
I'm not sure it serves the same purpose. Ideally this should just be the root route url passed in.

Actually, I know why, I did it - in case we do need a 2-or-more-step update flow. In which case, the lack of explicit `next` would probably suffice.

Reply:
But in those cases that can be handled within that specific controller. It would be better to keep it simple for now and reduce the amount of new "update" specific implementations of things.

Reply:
There only 'update' specific thing now is `updateComponent` and that's now handled by the update controller